package.json

```diff
@@ -1136,45 +1185,56 @@
 				"description": "%copilot.toolSet.editing.description%",
 				"icon": "$(pencil)",
 				"tools": [
-					"createFile",
 					"createDirectory",
+					"createFile",
+					"editFiles",
 					"editNotebook",
-					"newJupyterNotebook",
-					"editFiles"
+					"newJupyterNotebook"
 				]
 			},
 			{
-				"name": "runNotebooks",
-				"description": "%copilot.toolSet.runNotebook.description%",
-				"icon": "$(notebook)",
+				"name": "read",
+				"description": "%copilot.toolSet.read.description%",
+				"icon": "$(eye)",
 				"tools": [
-					"runCell",
-					"getNotebookSummary",
-					"readNotebookCellOutput"
+					"readFile",
+					"usages"
 				]
 			},
 			{
 				"name": "search",
 				"description": "%copilot.toolSet.search.description%",
 				"icon": "$(search)",
 				"tools": [
+					"codebase",
 					"fileSearch",
-					"textSearch",
 					"listDirectory",
-					"readFile",
-					"codebase",
-					"searchResults"
+					"searchResults",
+					"textSearch"
 				]
 			},
 			{
-				"name": "new",
```

Comment:
This toolset is driving a specific scenario for `#new` which helps the user set up a new workspace from scratch. I've never been crazy about how it works but we should either keep it or talk through it with Bhavya. And is `runVscodeCommand` missing from agent mode with this change?

Reply:
Got it, yes let me chat with Bhavya to make sure we're on the same page.

> is runVscodeCommand missing from agent mode with this change?

I renamed it to `vscode/runCommand` to avoid redundancy, but maybe we should keep it as is?

Reply:
Maybe '#new' should better be a slash-command `/new`. As a slash command it could make sure that it has all tools needed. And give extra context on how to create things. FYI @bhavyaus